.github/workflows/vulnerability-scanner-tests.yml

```diff
@@ -0,0 +1,96 @@
+name: Vulnerability Scanner
+
+on:
+  pull_request:
+    # Pull request events
+    types: [synchronize, opened, reopened, ready_for_review]
+    
+    # Path filtering
+    paths:
+      - ".github/workflows/vulnerability-scanner-tests.yml"
+      - "src/shared_modules/router/**"
+      - "src/shared_modules/content_manager/**"
+      - "src/shared_modules/indexer_connector/**"
+      - "src/wazuh_modules/vulnerability_scanner/**"
+      - "src/wazuh_modules/wazuh-http-request/**"
```

Comment:
Is this unused?

Reply:
It prevents the workflow from running when files other than the ones in te list are modified

Reply:
Aaah okay, is a built-in [feature](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-files-changed-in-a-pull-request). Nice.